procedure requests/README.md

```diff
@@ -0,0 +1,134 @@
+
+### Unmapped fields
+
+The following Procedure Request fields are not currently populated by the adaptor:
+
+- performer
+- reasonCode
+- reasonReference
+
+<details>
+    <summary>Example JSON</summary>
+
+```
+{
+    "resource": {
+        "resourceType": "ProcedureRequest",
+        "id": "3316531F-5705-424C-9E1A-EE694FB411B4",
+        "meta": {
+            "profile": [
+                "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-ProcedureRequest-1"
+            ]
+        },
+        "identifier": [
+            {
+                "system": "https://PSSAdaptor/2167888433",
+                "value": "3316531F-5705-424C-9E1A-EE694FB411B4"
+            }
+        ],
+        "status": "active",
+        "intent": "plan",
+        "code": {
+            "coding": [
+                {
+                    "extension": [
+                        {
+                            "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid",
+                            "extension": [
+                                {
+                                    "url": "descriptionId",
+                                    "valueId": "282653015"
+                                },
+                                {
+                                    "url": "descriptionDisplay",
+                                    "valueString": "Medication review"
+                                }
+                            ]
+                        }
+                    ],
+                    "system": "http://snomed.info/sct",
+                    "code": "182836005",
+                    "display": "Review of medication"
+                }
+            ],
+            "text": "Medication review"
+        },
+        "subject": {
+            "reference": "Patient/180b44bf-31d8-407b-b8ca-994a3f4a226c"
+        },
+        "context": {
+            "reference": "Encounter/3BFD78DE-03BF-44FD-96BC-CDF3DB2CC039"
+        },
+        "occurrenceDateTime": "2011-01-15",
+        "authoredOn": "2010-01-15T10:06:46+00:00",
+        "requester": {
+            "agent": {
+                "reference": "Practitioner/2D70F602-6BB1-47E0-B2EC-39912A59787D"
+            }
+        }
+    }
+}
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+A Procedure Request is mapped to a Plan Statement.
+
+| Mapped to (XML HL7)                              | Mapped from (JSON FHIR / other source )                                                                                                                              |
+|--------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| PlanStatement / id \[@root]                      | `ProcedureRequest.id`                                                                                                                                                |
```

Comment:
The gp2gp adaptor generates a new id, the procedureRequest.id is only used the check if the resource id has been mapped